MapboxDirectionsTests/RouteOptionsTests.swift

```diff
@@ -32,6 +32,7 @@ class RouteOptionsTests: XCTestCase {
         XCTAssertEqual(unarchivedOptions.locale, options.locale)
         XCTAssertEqual(unarchivedOptions.includesSpokenInstructions, options.includesSpokenInstructions)
         XCTAssertEqual(unarchivedOptions.distanceMeasurementSystem, options.distanceMeasurementSystem)
+        XCTAssertEqual(unarchivedOptions.roadClassesToAvoid, options.roadClassesToAvoid)
```

Comment:
Update `RouteOptions.testInstance` to specify `roadClassesToAvoid`. Otherwise, this assertion is meaningless.